triplea_maps.yaml

```diff
@@ -938,28 +938,29 @@
   description: |
     <br>This is a MAP SKIN for "Diplomacy"
     <br>A very basic style.  I made this one for people who hate the cursive writing of the other 3 skins.  It has a simple font, no textures, and no cursive.
-  version: 1
+  version: 1.7
```

Comment:
This version number will trigger the versions in the map download. That is the only place it is used, to show a map as out of date for download.